integration-tests/gradle/src/main/resources/basic-composite-build-extension-project/application/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins{
+    id "java"
+    id "io.quarkus"
+}
+
+
+
+group 'io.quarkus.test.application'
+version '1.0-SNAPSHOT'
+
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    implementation enforcedPlatform("${quarkusPlatformGroupId}:${quarkusPlatformArtifactId}:${quarkusPlatformVersion}")
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
```

Comment:
Looks like I missed these versions. The junit dependencies are included in the quarkus-bom and they are 5.9.1 now. Could you simply remove the versions?